ios/ci_scripts/ci_post_clone.sh

```
@@ -1,5 +1,9 @@
 #!/bin/sh
 
+# Fail fast in CI. Without this, `pod install` / `npm install` failures can be silently ignored,
+# and Xcode later fails with missing `Pods-*.xcconfig` / `.xcfilelist` files during the Analyze/Build phase.
+set -euo pipefail
```

Shellcheck (0.11.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)